api/pkg/backend/service.go

```diff
@@ -694,7 +694,7 @@ func (s *APIService) UpdateTier(request *restful.Request, response *restful.Resp
 	log.Info("Check backends should exists before removing")
 	backendsNotExist := utils.ResourcesCheckBeforeRemove(res.Tier.Backends, updateTier.DeleteBackends)
 	if len(backendsNotExist) > 0 {
-		errMsg := fmt.Sprintf("failed to update tier because delete backends: %v doesnt exist in tier", backendsNotExist)
+		errMsg := fmt.Sprintf("failed to update tier because delete backends: %v doesnt exist in service plan", backendsNotExist)
```

Comment:
failed to update tier ==>failed to update service plan

Reply:
changed